src/drc/api/serializers.py

```diff
@@ -131,6 +137,23 @@ def validate_indicatie_gebruiksrecht(self, indicatie):
             )
         return indicatie
 
+    def validate(self, attrs):
```

Comment:
I think this should be `def validate_lock` to limit it to the field `lock`.

Additionally, I'm missing the check that the `lock` may not be set/provided when creating the document, because then you'd start with a locked document which doesn't make sense.

Reply:
About `validate_lock`. As far as I know this method could be used only of there is a 'lock' field in request. So if I patch a document without specifying `lock` `validate_lock` won't be run.

Reply:
oh yeah, PATCH is a weird beast, nice catch!